helm/superset/templates/init-job.yaml

```diff
@@ -83,6 +83,18 @@ spec:
         command: {{  tpl (toJson .Values.init.command) . }}
         resources:
           {{- toYaml .Values.init.resources | nindent 10 }}
+    {{- with .Values.init.nodeSelector }}
```

Comment:
The current values for these apply to all deployments, do we really need one specific for the init job?

Reply:
You are true, I'm going to fix it